lib/teracy-dev/extension/manager.rb

```diff
@@ -20,17 +20,24 @@ def install(extensions)
         @logger.debug("extensions: #{extensions}")
         timer_start = Time.now
         extensions.each do |extension|
-          sync(extension)
-          validate(extension)
+          if Util.true?(extension['enabled'])
+            sync(extension)
+            validate(extension)
+          end
         end
+
+        extention_manifest_list = extensions.map { |x| Manager.manifest(x) if Util.true?(x['enabled'])}.compact
+        extensions.each do |extension|
+          validate_dependencies(extension, extention_manifest_list) if Util.true?(extension['enabled'])
+        end
+
```

Comment:
we should loop this 1 time:
```
extention_manifest_list = ..
extensions.each do |extension|
    sync(extension)
    validate(extension, extention_manifest_list)
end
```

Reply:
We need to finish sync all extension before do version check. For example i require V5 but current on local repo is V4 remote repo is v5. manifest list must load after sync too